paseos/actors/ground_station_actor.py

```diff
@@ -0,0 +1,23 @@
+from loguru import logger
+import pykep as pk
+
+from paseos.actors.base_actor import BaseActor
+
+
+class GroundstationActor(BaseActor):
+    """This class models a groundstation actor."""
+
+    def __init__(
+        self, name: str, position, velocity, epoch: pk.epoch, central_body: pk.planet
+    ) -> None:
+        """Constructor for a groundstation actor
+
+        Args:
+            name (str): Name of this actor
+            position (list of floats): [x,y,z]
```

Comment:
It looks odd that Groundstation has a "velocity". I guess you are doing that because our (0,0,0) is not necessarily the ground station. Maybe, add a comment.

Reply:
Everything is in central body frame, the ground station is "orbiting" it with the Earth's (or other body's) rotation as velocity and position relative to the center. Added a comment to clarify